### PR TITLE
fix crash

### PR DESCRIPTION
**Pull request by xxFLORII**
This should fix this error:

`2022-12-26 [10:03:53.547] [Server thread/CRITICAL]: TypeError: "tedo0627\redstonecircuit\block\mechanism\BlockCommand::dispatch(): Return value must be of type bool, null returned" (EXCEPTION) in "plugins/RedstoneCircuit.phar/src/tedo0627/redstonecircuit/block/mechanism/BlockCommand" at line 313
--- Stack trace ---
  #0 plugins/RedstoneCircuit.phar/src/tedo0627/redstonecircuit/block/mechanism/BlockCommand(228): tedo0627\redstonecircuit\block\mechanism\BlockCommand->dispatch()
  #1 plugins/RedstoneCircuit.phar/src/tedo0627/redstonecircuit/block/mechanism/BlockCommand(184): tedo0627\redstonecircuit\block\mechanism\BlockCommand->execute()
  #2 pmsrc/src/world/World(922): tedo0627\redstonecircuit\block\mechanism\BlockCommand->onScheduledUpdate()
  #3 pmsrc/src/world/World(879): pocketmine\world\World->actuallyDoTick(int 14190)
  #4 pmsrc/src/world/WorldManager(348): pocketmine\world\World->doTick(int 14190)
  #5 pmsrc/src/Server(1818): pocketmine\world\WorldManager->tick(int 14190)
  #6 pmsrc/src/Server(1700): pocketmine\Server->tick()
  #7 pmsrc/src/Server(1061): pocketmine\Server->tickProcessor()
  #8 pmsrc/src/PocketMine(339): pocketmine\Server->__construct(object BaseClassLoader#3, object pocketmine\utils\MainLogger#4, string[34] /home/clientservers/server-836907/, string[42] /home/clientservers/server-836907/plugins/)
  #9 pmsrc/src/PocketMine(362): pocketmine\server()
  #10 pmsrc(11): require(string[73] phar:///home/serversoftwares/PocketMine_1.19.50_2.phar/src/PocketMine.php)
--- End of exception information ---`

### CODE REVIEW

**Comment by tedo0627** [comment]
I did not feel the need to change the return value of the dispatch function.
I would change it like this.
In function dispatch of class BlockCommand
```php
$successful = $target->execute($this, $sentCommandLabel, $args);
```
↓
```php
$result = $target->execute($this, $sentCommandLabel, $args);
if (is_bool($result)) $successful = $result;
```

And, I don't think ```RegisterMappingsAsync``` class is necessary either.

**Comment by xxFLORII** [comment]
i‘ve added `RegisterMappingsAsync` to pass all poggit checks.

I will also implement the other suggestion immediately.

**Comment by tedo0627** [comment]
The poggit check is due to a phpstan error and can be ignored.
```
Severity: Error

PHPStan Error
An error occurred while running PHPStan

PHPStan failed with ID 'phpstan-b785-38ad42bd', contact support with the ID for help if this persists.
```
This error hasn't gone away for a long time, but I'm too lazy to report it, so I'm leaving it alone.

**Comment by xxFLORII** [comment]
should be done now.

File: src/tedo0627/redstonecircuit/RedstoneCircuit.php [hunk 1]
```diff
@@ -185,7 +185,7 @@ public function onLoad(): void {
 
         self::registerMappings();
         $this->getServer()->getAsyncPool()->addWorkerStartHook(function (int $worker): void {
-            $this->getServer()->getAsyncPool()->submitTaskToWorker(new class extends AsyncTask {
+            $this->getServer()->getAsyncPool()->submitTaskToWorker(new class extends AsyncTask{
```
Comment (tedo0627) [inline src/tedo0627/redstonecircuit/RedstoneCircuit.php]: I'm concerned about the loss of space here.

**Review by tedo0627 — COMMENTED** [review]
No problem except for the comment part.

**Comment by xxFLORII** [comment]
pushed

**Comment by tedo0627** [comment]
Thank you

**Pull request merged**